packages/client/hmi-client/src/components/models/tera-stratified-model-configuration.vue

```diff
@@ -175,6 +212,15 @@ function openValueModal(id: string, configIndex: number) {
 	}
 }
 
+function resetCellEditing() {
+	const row = { name: false };
```

Comment:
This data structure is a little weird, "name" is typically thought of as a string no? - what are we trying to convey?